src/linux_mcp_server/connection/ssh.py

```diff
@@ -286,7 +287,7 @@ async def execute_command(
     # Route to remote execution if host is provided
     if host:
         if not username:
-            username = getpass.getuser()
+            username = os.getenv("LINUX_MCP_USER", getpass.getuser())
```

Comment:
Is this a reasonable fallback? In the container, it's going to fall back to `mcp`, a completely non-sensical choice. Wouldn't it be more prudent to just fail unequivocally if `LINUX_MCP_USER` is undefined?